src/main/java/pl/szczesniak/dominik/whattowatch/movies/domain/WatchedMovie.java

```diff
@@ -0,0 +1,21 @@
+package pl.szczesniak.dominik.whattowatch.movies.domain;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+import pl.szczesniak.dominik.whattowatch.movies.domain.model.MovieId;
+import pl.szczesniak.dominik.whattowatch.movies.domain.model.MovieTitle;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.UserId;
+
+@ToString
+@RequiredArgsConstructor
+public class WatchedMovie {
+
```

Comment:
Można by się było zastanowić nad osobnym id dla watched movie. Wydaje mi się, że nie jest potrzebny, ale trzeba się liczyć z tym, że jeśli movieId jest identyfikatorem watched movie to musi być unikalny, więc wtedy nie będzie się dało obejrzeć filmu z listy kilka razy